### DISCUSSION TITLE
Which api model to use to list categories and products

### DISCUSSION THREAD

**Discussion by Usama99786** [amzn/selling-partner-api-models#346] [Q&A]
I want to list the categories and products of those categories in my android app using the SP Apis. I want to make it like the Amazon Shopping app. Later on, i also want to implement functionalities like add to cart, cart listing and place orders. I am very new to SP apis, so i'll really appreciate some help. Thanks

**Comment by SylvieManx** [comment]
I think you don't understand what the functions of SPAPI are, and the SPAPI
you requested is not supported.

Usama99786 ***@***.***> 于2023年12月15日周五 14:28写道：

**Comment by ericblade** [comment]
Agree with @401009641 .. i don't entirely understand your question, but whatever you're asking, spapi does not involve any of that.

**Comment by rodrifed** [comment]
Hi @Usama99786,

The Selling Partner API (SP-API) is a REST-based API that helps **Amazon selling partners** programmatically access their data on orders, shipments, payments, and much more.
From your description, I understand yours is a different use case that is not supported by the SP-API.

Best regards,
Federico
Selling Partner Developer Services
